voltdb/README.md

```diff
@@ -128,7 +128,7 @@ Need help? Contact [Datadog support][10].
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: https://voltdb.com
-[2]: https://docs.datadoghq.com/agent/kubernetes/integrations/
```

Comment:
Ah didn't realize that `[2]` was being used in `see the [Autodiscovery Integration Templates][2]`. It's also being used in `The VoltDB check is included in the [Datadog Agent][2] package.`, so I'll just set a new link instead of changing this one.